subprojects/build-cache-http/src/integTest/groovy/org/gradle/caching/http/internal/HttpBuildCacheServiceTest.groovy

```diff
@@ -94,15 +94,15 @@ class HttpBuildCacheServiceTest extends Specification {
     }
 
     def "can cache artifact with redirect"() {
```

Comment:
The name of the test should reflect what's happening.

Reply:
fixed.